queue/src/main/java/org/killbill/bus/api/PersistentBusConfig.java

```diff
@@ -92,4 +92,16 @@
     @Default("bus_events_history")
     @Description("Bus events history table name")
     public abstract String getHistoryTableName();
+
+    @Override
+    @Config("org.killbill.persistent.bus.${instanceName}.reapThreshold")
+    @Default("5m")
```

Comment:
I would change the default to at least `10m` (i know i had suggested `5m` on the ticket); Re-reading the code, `5m` is also the default time entries are being claimed, so this `reapThreshold` default value should be greater than the default `claim.time`.

Reply:
👍 

Should we also ensure in the code that `claim.time < reapThreshold` and override `reapThreshold` if its mis-configured (with a `WARN` log)?